bench/tests/test_run_repo_build.py

```
@@ -0,0 +1,315 @@
+                str(bench_home),
+                "run",
+                "--native-only",
+                "--dry-run",
+                "--no-resolve",
+                "--skip-prewarm",
+                "--runs",
+                "1",
+                "--session",
+                "test-session",
+            ]
+            proc = subprocess.run(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True, check=False)
```

Ruff (0.15.13)

[error] `subprocess` call: check for execution of untrusted input

(S603)